test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1350,21 +1368,46 @@ def get_cgroup_job_dir(self, subsys, jobid, host):
         Returns path of subsystem for jobid
         """
         basedir = self.paths[subsys]
-        # one of these should exist depending on platform
-        # That last option should never exist, but let us
-        # leave it in there
-        jobdirs = [os.path.join(basedir, 'pbs', jobid),
-                   os.path.join(basedir, 'pbs.service/jobid', jobid),
-                   os.path.join(basedir, 'pbs_jobs.service/jobid', jobid),
-                   os.path.join(basedir, 'pbs.slice',
-                                'pbs-%s.slice' % systemd_escape(jobid)),
-                   os.path.join(basedir, 'pbs',
-                                'pbs-%s.slice' % systemd_escape(jobid))]
+        # One of these should exist depending on platform.
+        #
+        # Entries for older hooks deleted. I am leaving them as comments should
+        # this be used to test old hooks, but DO NOT put them at the top
```

Comment:
Don't leave code as comments.  If we need to test older hooks, we can dig it out of git.   That's what code revision control is for.

Reply:
OK -- I'll simply specify the paths here, but not the code.